styles/flow/mod.css

```diff
@@ -14,31 +14,36 @@ pre {
   padding: 1rem;
   border-radius: var(--bd-radius);
   background: var(--bg-muted);
-  font-size: .85rem;
-  line-height: 1.45;
+  font-size: 0.9rem;
+  line-height: 1.6;
+  box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.1);
 }
 
 pre > code {
   overflow: visible;
   padding: 0;
   border-radius: 0;
   background: transparent;
-  font-size: 100%;
+  font-size: inherit;
 }
 
 /* Quotation block */
 blockquote {
-  padding: .25rem 1rem;
-  border-left: .25rem solid var(--bd-muted);
-  margin: 0 0 1rem;
+  padding: 0.5rem 1.5rem;
+  border-left: 0.25rem solid var(--primary-color);
+  margin: 0 0 1.5rem;
   color: var(--muted);
+  background: var(--bg-light);
```

Comment:
The variable `--bg-light` doens't exist, also I'm not convinced a blockquote should have a background color